tests/trading/test_topics.py

```diff
@@ -49,6 +49,6 @@ def test_topics(fundedRepFixture):
         topics.getTopicByOffset(0): topics.getPopularityByOffset(0),
         topics.getTopicByOffset(1): topics.getPopularityByOffset(1),
     } == {
-        bytesToLong('Politics'.ljust(32, '\x00')): 10,
-        bytesToLong('Sports'.ljust(32, '\x00')): fix('2.4'),
+        'Politics'.ljust(32, '\x00'): 10,
+        'Sports'.ljust(32, '\x00'): fix('2.4'),
```

Comment:
I'm curious about this. Was the `bytesToLong` never necessary? Or is Solidity's behavior different than Serpent in this case?

Reply:
Solidity's behavior is different in this case.